Backend/dev/migrations/dynamic-offer-driver-app/0355-whitelist-new-queries.sql

```diff
@@ -1,21 +1,17 @@
 ALTER TABLE atlas_driver_offer_bpp.white_list_org
-ADD COLUMN domain character varying(255) default NULL;
-
-INSERT INTO atlas_driver_offer_bpp.white_list_org (id, subscriber_id, type, domain)
-SELECT
-    (md5(random()::TEXT || clock_timestamp()::TEXT || random()::TEXT || random()::TEXT || random()::TEXT || random()::TEXT)::UUID) AS id,
-    subscriber_id,
-    'BAP' AS type,
-    'MOBILITY' AS domain
-FROM atlas_driver_offer_bpp.white_list_org
-WHERE type = 'APP';
+ADD COLUMN domain character varying(255) NULL;
 
 UPDATE atlas_driver_offer_bpp.white_list_org
-SET domain = 'MOBILITY_OLD'
-WHERE type = 'APP';
+SET domain = 'MOBILITY';
 
 ALTER TABLE atlas_driver_offer_bpp.white_list_org
 ALTER COLUMN domain SET NOT NULL;
 
 ALTER TABLE atlas_driver_offer_bpp.black_list_org
-ADD COLUMN domain character varying(255) NOT NULL;
\ No newline at end of file
+ADD COLUMN domain character varying(255) NOT NULL;
+
```

Comment:
Add comment to run these queries only after successful release to maintain backward compatibility in case if we need to revert